frontend/packages/dev-console/src/components/topology/operators/TopologyOperatorBackedResources.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import OperatorBackedOwnerReferences from '@console/internal/components/utils';
+import { OverviewItem } from '@console/shared';
+import { PodsOverview } from '@console/internal/components/overview/pods-overview';
+import { BuildOverview } from '@console/internal/components/overview/build-overview';
+import { NetworkingOverview } from '@console/internal/components/overview/networking-overview';
+import { TopologyDataObject } from '../topology-types';
+
+type TopologyOperatorBackedResourcesProps = {
+  item: TopologyDataObject;
+};
+
+const TopologyOperatorBackedResources: React.FC<TopologyOperatorBackedResourcesProps> = ({
+  item,
+}) => {
+  const combinedResources = item.groupResources.reduce<OverviewItem>(
+    (acc, val) => {
+      _.keys(acc).forEach((key) => {
+        Array.isArray(acc[key])
+          ? (acc[key] = [...acc[key], ...val.resources[key]])
+          : (acc[key] = { ...acc[key], ...val.resources[key] });
+      });
+      return acc;
+    },
+    { buildConfigs: [], pods: [], obj: {}, routes: [], services: [] },
+  );
+  const { buildConfigs, pods, obj, routes, services } = combinedResources;
+  combinedResources.isOperatorBackedService = true;
```

Comment:
can we have it as default value is accumulator for reducer? WDYT

Reply:
Updated.